icssc/peterportal-client
# pnpm

## Description
- switched to pnpm
  - updated scripts
  - updated actions workflows
  - deleted package-locks
  - added pnpm-workspace file
  - updated readme
- updated a few packages
- fixed lint/tsc errors


## Steps to verify/test this change:

- [ ] Verify changes work as expected on staging instance
  1. delete node_modules folders
  2. run `pnpm install`
  3. test dev `pnpm dev`
  4. test making a commit

## Final Checks:

- [ ] Verify successful deployment

(optional)


## Issues



Closes #
